### PR TITLE
Add new canadian provinces

### PR DESCRIPTION
**Pull request by j949wang**
## What?

Add new Canadian provinces to documentation

## Why?

With the introduction of the international regions v1 pipeline. We are starting to introduce new regions.

## Screenshots (optional)

Screen Shot 2023-03-03 at 15 41 02
Screen Shot 2023-03-03 at 15 40 51

### CODE REVIEW

File: docs/regions/states.mdx [hunk 1]
```diff
@@ -2,60 +2,73 @@
...
-`WA` | `Washington`
-`WI` | `Wisconsin`
-`WV` | `West Virginia`
-`WY` | `Wyoming`
+Code | Country | Name  
+--- |  --- | ---
+`AK` | `United States` | `Alaska`
+`AL` | `United States` | `Alabama`
+`AR` | `United States` | `Arkansas`
+`AB` | `Canada` | `Alberta`
```
Comment (david-goodfellow) [inline docs/regions/states.mdx:14]: Up to product but thinking we may want to group them by country. Like put all United States then all canada

File: docs/regions/states.mdx [hunk 2]
```diff
@@ -2,60 +2,73 @@
 title: States
 ---
 
-[Regions](/regions) supports state detection in the US only. Each state has a `name` and a unique 2-letter `code`.
+[Regions](/regions) supports state detection in the US and Canada only. Each state has a `name`, a unique 2-letter `code` and the residing `country`.
 
 ## State list
```
Comment (jsani-radar) [inline docs/regions/states.mdx:7]: not a big fan of "State list" as the top-level header and only href on the page

let's remove this and split it out so we have:
## United states
[table with US states]

## Canada
[table with Canadian provinces]

**Review by jsani-radar — COMMENTED** [review]
small comment about formatting, content lgtm

will be 👍🏽 on next pass

File: docs/regions/states.mdx [hunk 3]
```diff
@@ -2,60 +2,73 @@
 title: States
 ---
 
-[Regions](/regions) supports state detection in the US only. Each state has a `name` and a unique 2-letter `code`.
+[Regions](/regions) supports state detection in the US and Canada only. Each state has a `name`, a unique 2-letter `code` and the residing `country`.
```
Comment (jsani-radar) [inline docs/regions/states.mdx:5]: nit: comma usage should be consistent with top-level Regions page

File: docs/regions.mdx [hunk 4]
```diff
@@ -59,7 +59,7 @@ Regions can also be used to allowlist or blocklist location updates in specific
 
 ## States
 
-Regions supports state detection in the US only. Each state has a `name` and a unique 2-letter `code`. View the [full list of states](/regions/states).
+Regions supports state detection in the US and Canada only. Each state has a `name`, a unique 2-letter `code`, and the residing `country`. View the [full list of states](/regions/states).
```
Comment (jsani-radar) [inline docs/regions.mdx:62]: nit: the uniqueness is on the 2-letter code AND country, correct?

File: docs/regions/states.mdx (hunk 2 above)
Comment (j949wang) [reply docs/regions/states.mdx:7]: Sounds good, should we still keep the `country` column?

Comment (jsani-radar) [reply docs/regions/states.mdx]: country column is fine, but let's make that last so it's easy to see the correspondence between the 2-letter code and name

**Review by jsani-radar — APPROVED** [review]

**Pull request merged**